From f988690d1cd2d60e7f7db5d47593adb69afdb202 Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Wed, 16 Nov 2016 17:31:22 +0200
Subject: [PATCH 0590/1345] dt-bindings: net: add comphy description to
 marvell-pp2x.txt

commit  688e4fc4db8cbe692942d2a69d1184dc18790c4a from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

- add comphy description
- update example with comphy

Change-Id: I0e7776579975497b2c91d0383f45736129175f62
Reviewed-on: http://vgitil04.il.marvell.com:8080/33858
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../devicetree/bindings/net/marvell-pp2x.txt       |    6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Documentation/devicetree/bindings/net/marvell-pp2x.txt b/Documentation/devicetree/bindings/net/marvell-pp2x.txt
index a72114b..5c982ab 100644
--- a/Documentation/devicetree/bindings/net/marvell-pp2x.txt
+++ b/Documentation/devicetree/bindings/net/marvell-pp2x.txt
@@ -39,6 +39,8 @@ emac required properties:
 - port-id: Should be '0', '2', or '3' for ethernet macs, references the mac-id (GOP port-id).
 - phy-mode: See ethernet.txt file in the same directory
 - phy-speed: For 2.5Gbits SGMII port 2500 phy-speed should be set(used only for 2.5Gbits SGMII port)
+- phy-names: Used in MAC's connected to comphy. See phy-mvebu-comphy.txt in the phy directory
+- phys: Used in MAC's connected to comphy. Comphy index, see  phy-mvebu-comphy.txt in the phy directory
 
 emac optional properties:
 ========================
@@ -53,6 +55,8 @@ gop {
 		mac-address = [00 00 00 00 00 01];
 		port-id = <0>; /* gop_port_id */
                 phy-mode = "sgmii"; /* lane-2 */
+		phys = <&comphy0 2 COMPHY_SGMII>;
+		phy-names = "comphy";
         };
 	emac2: mac2 {
 		interrupts = <GIC_SPI 35 IRQ_TYPE_LEVEL_HIGH>;  /* Link IRQ */
@@ -60,6 +64,8 @@ gop {
 		port-id = <2>; /* gop_port_id */
 		phy-mode = "sgmii"; /* lane-0 */
 		phy-speed = <2500>;
+		phys = <&comphy0 0 COMPHY_HS_SGMII>;
+                phy-names = "comphy";
                 phy = <&phy2>;
 	};
 	emac3: mac3 {
-- 
1.7.9.5

